tests/cli/test_transfer.md

```diff
@@ -0,0 +1,106 @@
+# Tests Transfer tutorial
+
+## Init
+
+```sh
+$ [ -d transfer ] || atomkraft init transfer
+...
+$ [ -d transfer ] && echo Project directory created successfully
+Project directory created successfully
+```
+
+## Model and traces
+
+```sh
+$ curl -Lo transfer/models/transfer.tla https://raw.githubusercontent.com/informalsystems/atomkraft/dev/examples/cosmos-sdk/transfer/transfer.tla
+...
+```
+
+<!-- $MDX dir=transfer -->
+```sh
+$ atomkraft model apalache info
+...
+Apalache JAR file exists and its version is 0.25.10
+...
```

Comment:
If `atomkraft init` is executed for the first time, `apalache` is not present at this point.